gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -178,12 +178,8 @@ def create_postgres_root_user
     end
 
     def create_postgres_database
-      run_as_postgres("createdb -E utf8 -O #{username} #{database}")
-    end
-
-    # some overlap with PostgresAdmin
-    def run_as_postgres(cmd)
-      AwesomeSpawn.run("su", :params => {"-" => nil, nil => "postgres", "-c" => "#{PostgresAdmin.scl_enable_prefix} \"#{cmd}\""})
+      conn = PG.connect(:user => "postgres", :dbname => "postgres")
+      conn.exec("CREATE DATABASE #{database} OWNER #{username} ENCODING 'utf8'")
```

Comment:
also, this assumes that the current user is allowed to connect to the database as postgres. before, we were linux user `postgres` so they could go in.

Think our `pg_hba.conf` and `pg_map.conf` are configured to allow this, but just putting that out there.

nice change